scripts/emr/terraform/variables.tf.json

```diff
@@ -0,0 +1,33 @@
+{ 
```

Comment:
Why is this a json file instead of an actual terraform file (`.tf`)?

Reply:
To parse `json` we can use `jq`, but to use `.tf` we need to parse it somehow ugly.

Reply:
Why do we need to parse a file?  What is that in service of?

Reply:
To parse `KEY_PAIR_FILE` from file, if it's necessary. `pem` file is required to have a convient way to access master node.

Reply:
Ok, to answer more complete: there is set of useful commands in [this Makefile](https://github.com/pomadchin/geotrellis/blob/c2b3443fd68fe5c5fcb70f296df00dc51d3d5dcb/scripts/emr/Makefile#L29-L53), that's why all these things are required.

Reply:
Okay